Configuration/build.gradle

```diff
@@ -24,19 +24,6 @@ dependencies {
     testRuntimeOnly libs.slf4j.simple
 }
 
-sourceSets {
-    main {
-        resources {
-            srcDir 'src/main/java' exclude '**/*.java'
```

Comment:
There is only one non-test resource in Configuration/src, and that is lib-tests.prop in src/main/resources. Perhaps this should be in its own test-only artifact anyway?